src/src/com/microsoft/aad/adal/ADALError.java

```diff
@@ -515,8 +515,28 @@
     /**
      * Resource authentication challenge failure
      */
-    RESOURCE_AUTHENTICATION_CHALLENGE_FAILURE("Resource authentication challenge failure");
+    RESOURCE_AUTHENTICATION_CHALLENGE_FAILURE("Resource authentication challenge failure"),
 
+    /**
+     * Export of FID failure
+     */
+    FAIL_TO_EXPORT("Fail to export"),
+    
+    /**
+     * Import of FID failure
+     */
+    FAIL_TO_IMPORT("Fail to import"),
+        
+    /**
+     * Deserialization of FID failure
+     */
+    FAIL_TO_DESERIALIZE("Fail to deserialize"), 
```

Comment:
This should never happen, you always import only family refresh tokens otherwise blob should be null.